src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -264,49 +264,45 @@ DataView::ActionStatus LiveFunctionsDataView::GetActionStatus(
   }
 
   bool is_capture_connected = app_->IsCaptureConnected(capture_data);
-  if (action == kMenuActionDisassembly || action == kMenuActionSourceCode) {
-    return is_capture_connected ? ActionStatus::kVisibleAndEnabled
-                                : ActionStatus::kVisibleButDisabled;
-  }
-
-  if ((action == kMenuActionSelect || action == kMenuActionUnselect) && !is_capture_connected) {
-    return ActionStatus::kVisibleButDisabled;
-  }
 
   std::function<bool(uint64_t, const FunctionInfo&)> is_visible_action_enabled;
-  if (action == kMenuActionSelect) {
-    is_visible_action_enabled = [this](uint64_t /*instrumented_function_id*/,
-                                       const FunctionInfo& instrumented_function) {
-      return !app_->IsFunctionSelected(instrumented_function) &&
-             instrumented_function.IsFunctionSelectable();
+  if (action == kMenuActionDisassembly || action == kMenuActionSourceCode) {
+    is_visible_action_enabled = [is_capture_connected](uint64_t /*scope_id*/,
+                                                       const FunctionInfo& /*function_info*/) {
+      return is_capture_connected;
+    };
+
```

Comment:
I didn't put `SourceCode` and `Disassembly` here as I would like to return early for this case. We don't need to get `FunctionInfo` or checking the selected indices for these two actions.